faktorzehn/evendemy
# Adding github-actions-workflows to evendemy

## What has been done? 
There is now 3 new seperate github actions workflows in .github/workflows.
- One builds our backend and webapp and tests it
- One builds our backend and webapp and puts it into a docker container
- and the last one runs linting on both webapp and backend

## How to test
This is a bit tricky and i dont know how you could test it.
If you take a look into my fork and go to the actions tab you should see all the actions runs i have done

## ⚠️ Caution ⚠️ 
Don't merge this unless Ui Render in docker has been merged. This branch is based on the docker branch and might need to be rebased first before merging

## Thank you for taking the time to review this. Kind regards, Jeremy Sztavinovszki 😄
